### ISSUE TITLE
Possible false positives for CT 4.1.7.4.2 Product Linear Placement

### ISSUE THREAD

**Issue by RickBrice** [buildingSMART/ifc-gherkin-rules#124]
If I understand [CT 4.1.7.4.2 Product Linear Placement](https://standards.buildingsmart.org/IFC/RELEASE/IFC4_3/HTML/concepts/Product_Shape/Product_Placement/Product_Linear_Placement/content.html) correctly, IfcLinearPlacement.PlacementRelTo is the same IfcLocalPlacement as used to place the IfcLinearPositioningElement.

At the end of the attached file, three linear placements are defined.
`#279` IfcLinearPlacement.PlacementRelTo = `#22` which is the same placement for `#248` IfcAlignment.ObjectPlacement - This placement should be correct

`#287` IfcLinearPlacement.PlacementRelTo is a unique IfcLocalPlacement with the same values as `#22` but is a different placement - This seems to violate the CT but passes the validation service checks.

`#295` IfcLinearPlacement.PlacementRelTo is also a unique IfcLocalPlacement but with different values than `#22` - This seems to violate the CT as well but passes the validation service checks.

[FHWA_Bridge_Geometry_Alignment_Example.ifc.txt](https://github.com/buildingSMART/validate/files/13733158/FHWA_Bridge_Geometry_Alignment_Example.ifc.txt)

**Comment by aothms** [comment]
I think the original issue is definitely valid and actionable. I've tagged and moved it to the rule repo so that it can be picked up.

**Comment by civilx64** [comment]
@evandroAlfieri I propose we rename this to LIP001 since it corresponds to the Functional Part for Linear Placement.

**Comment by evandroAlfieri** [comment]
Agree @civilx64
And feel free to propose a more concise title ;) You can integrate the current one in the description

**Comment by aothms** [comment]
It's the same rule for LinearPlacement as well as GridPlacement. I agree that from a vendor / end-user perspective these should be separately positioned under `GDP` and `LIP` (I would probably name them `GPL` and `LPL` so that's a bit more clear that they are both `PL`acement related - but forgive me my obsessive side tracks).

This basically means that same thing we observed in "gcert". We end up copy pasting a lot, but that's fine. With the feature file / implementation divide and "plain-text-first" nature we kind of allow our selves to copy paste feature files as it's a bit like a configuration / orchestration file.

**Comment by evandroAlfieri** [comment]
Good that we can assign multiple tags. So, 
- let's use the generic one for the rule's code (i.e, `OJP` for object placement)
- then add `LIP` and `GDP` in the feature file, other than repeating `OJP`

@aothms I feel you on `GPL` and `LPL`, it was my first iteration too. Then, got stuck on the third subpart, Local Placement, being another `LPL` 
Any other idea?

**Comment by aothms** [comment]
> Good that we can assign multiple tags. So,

Not sure we should do this. I wasn't very clear. But I think it's better to have 1-many correspondence between rules and fparts. Also, the rule is a bit different in the selection, which is necessary for meaningfully process the `passed` part.

```
Given an IfcGridPlacement
Then magic from scott
``` 

```
Given an IfcLinearPlacement
Then magic from scott
``` 

> Local Placement, being another LPL
>
> Any other idea?

No 😢

**Comment by evandroAlfieri** [comment]
Two rules then, LIP001 and GDP001. I'll make the change in our backlog
